example/python/tx-example.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import print_function
+
+import sys
+# import iroha library from nested folder
+sys.path.insert(0, 'shared_model/bindings')
+import iroha
+
+import time
+import block_pb2
+import endpoint_pb2
+import endpoint_pb2_grpc
+import queries_pb2
+import grpc
+
+txbuilder = iroha.ModelTransactionBuilder()
+queryBuilder = iroha.ModelQueryBuilder()
+crypto = iroha.ModelCrypto()
+protoTxHelper = iroha.ModelProtoTransaction()
+protoQueryHelper = iroha.ModelProtoQuery()
+
+admin_priv = open("../admin@test.priv", "r").read()
+admin_pub = open("../admin@test.pub", "r").read()
+
+me_kp = crypto.convertFromExisting(admin_pub, admin_priv)
+
+current_time = int(round(time.time() * 1000)) - 10**5
+startCounter = 1
+creator = "admin@test"
+
+
+# build transaction
+tx = txbuilder.creatorAccountId(creator) \
+    .createdTime(current_time) \
+    .createDomain("ru", "user") \
+    .createAsset("dollar", "ru", 2).build()
+
+tx_blob = protoTxHelper.signAndAddSignature(tx, me_kp).blob()
+
+# create proto object and send to iroha
+
+proto_tx = block_pb2.Transaction()
+proto_tx.ParseFromString(''.join(map(chr, tx_blob)))
```

Comment:
bad example, why client needs to create proto by himself?
give him an option to receive proto file directly

Reply:
Current API does not allow it

Reply:
We should improve it later